### ISSUE TITLE
Big difference acc and val_acc

### ISSUE THREAD

**Issue by akirakitamura19** [alexander-rakhlin/CNN-for-Sentence-Classification-in-Keras#17]
Hello, I ran the this code but I got much worse result compared to the score author said.
Although I tried this code in both backends, tensorflow and Theano, both I got similar score.
Do you have any idea how to deal with the problem?
Thanks

![screenshot 2017-05-04 07 42 46](https://cloud.githubusercontent.com/assets/22701282/25683155/37baf966-309e-11e7-8a54-39bfed4561ce.png)

**Comment by alexander-rakhlin** [comment]
Hi,

Here is output with default settings

`Using Theano backend.
WARNING (theano.sandbox.cuda): The cuda backend is deprecated and will be removed in the next release (v0.10).  Please switch to the gpuarray backend. You can get more information about how to switch at this URL:
 https://github.com/Theano/Theano/wiki/Converting-to-the-new-gpu-back-end%28gpuarray%29

Using gpu device 0: GeForce GTX 980 (CNMeM is enabled with initial size: 90.0% of memory, cuDNN 5110)
Load data...
x_train shape: (25000, 400)
x_test shape: (25000, 400)
Model type is CNN-rand
Train on 25000 samples, validate on 25000 samples
Epoch 1/10
7s - loss: 0.5782 - acc: 0.6365 - val_loss: 0.3192 - val_acc: 0.8677
Epoch 2/10
7s - loss: 0.3063 - acc: 0.8735 - val_loss: 0.2846 - val_acc: 0.8827
Epoch 3/10
7s - loss: 0.2539 - acc: 0.8977 - val_loss: 0.2727 - val_acc: 0.8882
Epoch 4/10
7s - loss: 0.2334 - acc: 0.9064 - val_loss: 0.2768 - val_acc: 0.8868
Epoch 5/10
7s - loss: 0.2143 - acc: 0.9162 - val_loss: 0.2894 - val_acc: 0.8825
Epoch 6/10
7s - loss: 0.2065 - acc: 0.9189 - val_loss: 0.2821 - val_acc: 0.8848
Epoch 7/10
7s - loss: 0.2007 - acc: 0.9218 - val_loss: 0.2855 - val_acc: 0.8844
Epoch 8/10
7s - loss: 0.1926 - acc: 0.9234 - val_loss: 0.2929 - val_acc: 0.8839
Epoch 9/10
7s - loss: 0.1822 - acc: 0.9274 - val_loss: 0.2986 - val_acc: 0.8810
Epoch 10/10
7s - loss: 0.1835 - acc: 0.9292 - val_loss: 0.2924 - val_acc: 0.8800

Process finished with exit code 0
`

**Comment by akirakitamura19** [comment]
When I created a new versions of Python environment, this problem solved even backend is tensorflow. 

Thank you.
